packages/api/src/middleware/checkAnimalEntities.js

```diff
@@ -0,0 +1,70 @@
+import { Model, transaction } from 'objection';
+import { handleObjectionError } from '../util/errorCodes';
```

Comment:
My backend was crying that this isn't errorCodes.js

Reply:
Thank you! 🙏